help/en/html/tools/logixng/Tutorial/index.shtml

```diff
@@ -39,30 +39,49 @@
       a panel and the panels has names like "Chapter 1", that tells which chapter this
       panel is used for.</p>
 
-      <h3>Chapters</h3>
+      <h3>Standard Features</h3>
 
       <ul>
         <li><a href="chapter1.shtml">Chapter 1 - A simple LogixNG</a></li>
         <li><a href="chapter2.shtml">Chapter 2 - The clipboard</a></li>
         <li><a href="chapter3.shtml">Chapter 3 - Basic actions and expressions</a></li>
+        <li><a href="chapter4.shtml">Chapter 4 - Working with sockets</a></li>
+        <li><a href="chapter5.shtml">Chapter 5 - Action Items</a></li>
+        <li><a href="chapter6.shtml">Chapter 6 - Expression Items</a></li>
+      </ul>
+
+      <h3>Advanced Features</h3>
+
+      <ul>
+        <li><a href="chapter7.shtml">Chapter 7 - References</a></li>
+        <li><a href="chapter8.shtml">Chapter 8 - Local Variables</a></li>
+        <li><a href="chapter9.shtml">Chapter 9 - Formula</a></li>
+        <li><a href="chapter10.shtml">Chapter 10 - Modules</a></li>
+        <li><a href="chapter11.shtml">Chapter 11 - Tables</a></li>
       </ul>
 <hr>
 
+<hr>
+<hr>
+<hr>
+
+<h3>Ignore -- WIP</h3>
+
 <p>
 
 <p>
-<a href="chapter1.shtml">Chapter 1 - A simple LogixNG</a>
+<!-- <a href="chapter1.shtml">Chapter 1 - A simple LogixNG</a> -->
 <p>
 <a href="chapter2.html">Chapter 2 - The clipboard</a>
```

Comment:
I guess this line should also be commented out? Chapter 2 about the clipboard is in the section "Standard features" above.

Reply:
Most of the content in index.shtml will be removed in phase 3.  The redundant content will be dropped and the useful content moved to the proper chapters or archived for future chapters.